### PR TITLE
IGAPP-704: added linter rule for no-console and suppress specific con…

### PR DESCRIPTION
**Pull request by f1sh1918**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Review by Taggotty — APPROVED** [review]

File: .eslintrc.js [hunk 1]
```diff
@@ -72,6 +72,7 @@ module.exports = {
       files: ['*.ts', '*.tsx'],
       excludedFiles: ['*.spec.{ts,tsx}', '**/__mocks__/*.ts'],
       rules: {
+        'no-console': 2,
```
Comment (klinzo) [inline .eslintrc.js:75]:
```suggestion
        'no-console': 'error',
```

File: native/e2e/driver/driver.ts [hunk 2]
```diff
@@ -24,13 +24,15 @@ const getConfig = (): ConfigType | null => {
   const configName: string | null | undefined = process.env.E2E_CONFIG
 
   if (!configName) {
+    // eslint-disable-next-line no-console
```
Comment (klinzo) [inline native/e2e/driver/driver.ts:27]: You could also disable this rule for the whole file in this case I think.

File: native/src/components/EastereggImage.tsx [hunk 3]
```diff
@@ -49,6 +49,7 @@ class EastereggImage extends React.Component<PropsType, StateType> {
         })
       )
       .catch(e => {
+        // eslint-disable-next-line no-console
         console.error(e)
```
Comment (klinzo) [inline native/src/components/EastereggImage.tsx:53]: Would it be possible to exclude `console.error` from this rule?

File: native/src/components/FeedbackContainer.tsx [hunk 4]
```diff
@@ -134,6 +134,7 @@ const FeedbackContainer = (props: PropsType): ReactElement => {
       }
     })
     request().catch(err => {
+      // eslint-disable-next-line no-console
       console.log(err)
```
Comment (klinzo) [inline native/src/components/FeedbackContainer.tsx:138]:
```suggestion
      console.error(err)
```

**Review by klinzo — APPROVED** [review]

File: native/src/components/EastereggImage.tsx (hunk 3 above)
Comment (f1sh1918) [reply native/src/components/EastereggImage.tsx:53]: @klinzo 
    'no-console': ['error', { allow: ['error', 'warn'] }],
this should do the trick

**Pull request merged**